gravitee-apim-portal-webui-next/src/app/reset-password-confirmation/reset-password-confirmation.component.html

```diff
@@ -0,0 +1,96 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<form [formGroup]="resetPasswordConfirmationForm" (ngSubmit)="confirmResetPassword()">
+  <mat-card appearance="outlined" class="reset-password">
+    @if (!isSubmitted) {
+      <mat-card-header>
+        <mat-card-title i18n="@@resetPasswordConfirmationTitle">Reset password confirmation</mat-card-title>
+      </mat-card-header>
+      @if (!isTokenExpired) {
+        <mat-card-content class="reset-password__form">
+          <mat-form-field class="reset-password__form__input">
+            <mat-label i18n="@@resetPasswordConfirmationFirstname">First name</mat-label>
+            <input matInput formControlName="firstname" appearance="outlined" />
+          </mat-form-field>
+          <mat-form-field class="reset-password__form__input">
+            <mat-label i18n="@@resetPasswordConfirmationLastname">Last name</mat-label>
+            <input matInput formControlName="lastname" appearance="outlined" />
+          </mat-form-field>
+          <mat-form-field class="reset-password__form__input">
+            <mat-label i18n="@@resetPasswordConfirmationEmail">Email</mat-label>
+            <input matInput formControlName="email" appearance="outlined" />
+          </mat-form-field>
+          <mat-form-field class="reset-password__form__input">
+            <mat-label i18n="@@resetPasswordConfirmationPassword">Password</mat-label>
+            <input matInput formControlName="password" type="password" appearance="outlined" />
+            @if (resetPasswordConfirmationForm.controls['password'].hasError('required')) {
+              <mat-error i18n="@@resetPasswordConfirmationPasswordErrorRequired">Password is required</mat-error>
+            }
+          </mat-form-field>
+          <mat-form-field class="reset-password__form__input">
+            <mat-label i18n="@@resetPasswordConfirmationConfirmedPassword">Confirmed Password</mat-label>
+            <input matInput formControlName="confirmedPassword" type="password" appearance="outlined" (input)="update1()" />
+            @if (resetPasswordConfirmationForm.controls['confirmedPassword'].hasError('passwordMismatch')) {
```

Comment:
Since you are using a typed form, you can use some shortcuts 😉 
```suggestion
            @if (resetPasswordConfirmationForm.controls.confirmedPassword.hasError('passwordMismatch')) {
```